lib/modules/search.js

```diff
@@ -54,6 +54,8 @@ export function search(query?: string) {
 	SettingsNavigation.setUrlHash(module.moduleID, query);
 }
 
+const debouncedSearch = _.debounce(search, 250);
```

Comment:
I think `frameThrottle` is a more natural choice here since it won't use any more resources than available. In addition, debouncing prevents it from being "search as you type" since it will only search after stopping typing.